### PR TITLE
Remove manifests

### PR DESCRIPTION
**Pull request by robyoung**
The purpose of this change is to get the admin app service editing manifest files into the [digitalmarketplace-frameworks](https://github.com/alphagov/digitalmarketplace-frameworks) repo. As part of that there have needed to be quite a few changes to how service editing works in the admin app. These changes make service editing in the admin app the same as the supplier frontend.

This change introduces a lot of duplication between the admin and supplier apps in the service helpers module. This will be moved into [digitalmarketplace-utils](https://github.com/alphagov/digitalmarketplace-utils) in a later pull request.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "version": "0.0.1",
   "dependencies": {
     "jquery": "1.11.2",
-    "digitalmarketplace-frontend-toolkit": "git://github.com/alphagov/digitalmarketplace-frontend-toolkit#v10.0.0",
+    "digitalmarketplace-frontend-toolkit": "git://github.com/alphagov/digitalmarketplace-frontend-toolkit#v11.2.0",
     "govuk_template": "https://github.com/alphagov/govuk_template/releases/download/v0.14.1/jinja_govuk_template-0.14.1.tgz",
-    "digital-marketplace-ssp-content": "git://github.com/alphagov/digital-marketplace-ssp-content#15fcaeab5b24893a04bece14a56ff2fa425e5633",
+    "digital-marketplace-ssp-content": "git://github.com/alphagov/digital-marketplace-ssp-content#d47defa52ee51654a42b46d7d6e2bca8c8c14f50",
```
Comment (quis) [inline bower.json]: This can be https://github.com/alphagov/digitalmarketplace-frameworks/releases/tag/v0.0.1 now

Comment (robyoung) [inline bower.json]: Done

**Comment by TheDoubleK** [comment]
This has made the form fields really wide... seems like an odd side effect?  I've run all the pips, npms, frontend-build, etc. and this is how it looks (quite different from preview):
screen shot 2015-10-01 at 09 22 54

**Comment by quis** [comment]
@TheDoubleK Might need a rebase to bring in https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/129

**Comment by TheDoubleK** [comment]
Pricing updates don't work here: `{'priceMax': 'not_a_number', 'priceMin': 'not_a_number'}` - is this dependent on this API pull-request: https://github.com/alphagov/digitalmarketplace-api/pull/254?

File: app/main/helpers/service.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import re
```
Comment (TheDoubleK) [inline app/main/helpers/service.py]: I take it everything in this helper is a straight lift from supplier app? (i.e. I'm not going to review it right now)

**Comment by TheDoubleK** [comment]
Error messaging isn't quite right e.g. for an empty description at the moment (preview):
screen shot 2015-10-01 at 11 39 11

But on this branch:
screen shot 2015-10-01 at 11 39 27

File: app/templates/macros/toolkit_forms.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
+{% macro text(question_content, service_data, errors, question_number=None) -%}
```
Comment (TheDoubleK) [inline app/templates/macros/toolkit_forms.html]: Are these macros also something that is shared with supplier app?  Or are they different for admin?

File: config.py [hunk 4]
```diff
@@ -17,10 +17,10 @@ class Config(object):
     SESSION_COOKIE_PATH = '/admin'
     SESSION_COOKIE_HTTPONLY = True
     SESSION_COOKIE_SECURE = True
-    S3_DOCUMENT_BUCKET = os.getenv('DM_S3_DOCUMENT_BUCKET')
-    DOCUMENTS_URL = 'https://assets.dev.digitalmarketplace.service.gov.uk'
-    DM_DATA_API_URL = os.getenv('DM_DATA_API_URL')
-    DM_DATA_API_AUTH_TOKEN = os.getenv('DM_DATA_API_AUTH_TOKEN')
+    DM_S3_DOCUMENT_BUCKET = None
```
Comment (TheDoubleK) [inline config.py]: Nice to standardise these variable names :)

**Comment by robyoung** [comment]
@TheDoubleK @quis I have rebased on #129

**Comment by robyoung** [comment]
I have spoken to @TheDoubleK in real life about the service summary issue. It is due to the fact that we don't have the word count on the JSON schema (https://github.com/alphagov/digitalmarketplace-api/blob/master/json_schemas/services-g-cloud-5.json#L36). Adding it would be hard because any changes to G5 services with long summaries would fail.

File: app/templates/macros/toolkit_forms.html (hunk 3 above)
Comment (robyoung) [inline app/templates/macros/toolkit_forms.html]: Hey they are but we don't have a neat way of sharing them at the moment and given they don't contain a huge amount of the logic I've decided to leave them for the moment.

**Comment by TheDoubleK** [comment]
:+1:

**Pull request merged**